lighthouse-core/runner.js

```diff
@@ -186,7 +186,8 @@ class Runner {
     // resolution.
     .map(entry => {
       return /** @type {PerformanceEntry} */ ({
-        ...entry,
+        name: entry.name,
+        entryType: entry.entryType,
         duration: parseFloat(entry.duration.toFixed(2)),
         startTime: parseFloat(entry.startTime.toFixed(2)),
```

Comment:
can you reorder so it's (`startTime`, `name`, `duration`, `entryType`) just to prevent the churn in `sample_v2.json` :)

Reply:
We found that at the same time! Done.